docs/.vuepress/config.js

```diff
@@ -8,7 +8,12 @@ module.exports = {
     lineNumbers: true
   },
   themeConfig: {
-    lastUpdated: "Last Updated",
+    lastUpdated: true,
+    algolia: {
+      apiKey: '59f0e2deb984aa9cdf2b3a5fd24ac501',
```

Comment:
this is search-only API key (https://www.algolia.com/doc/guides/security/api-keys/?language=php#search-only-api-key), right? just want to make sure

Reply:
yes